generation/readme_update.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
When should this run? How about adding a check to ensure the README files have the annotations applied?

Reply:
Explained in comments.